_data/doc_categories.yml.v1.1

```diff
@@ -0,0 +1,10 @@
+# The order of the categories in this file (first-to-last) determines their
```

Comment:
Is this also auto-generated?

Reply:
Nope, this file was included mistakenly and is removed. Sorry about that.